samples/postman/README.md

```diff
@@ -0,0 +1,54 @@
+Postman CCG OAuth Token
+
+Description:
+
+This Postman collection contains pre-script request methods for VA API's using CCG OAuth. It's purpose is to generate an access token using the required test variables.
+
+Configuration:
+
+1. Import "Lighthouse OAuth Token.postman_collection.json" into your Postman workspace.
+2. Click on “Lighthouse OAuth Token” in the left column of Postman, to view the collection tabs for "Lighthouse OAuth Token".
+3. Click on "Variables" within Postman, to view pre set collection variables.
+4. Configure "Lighthouse OAuth Token" collection variables (aud, token_endpoint, clientId, privatePem) in Postman, assigning them their respective values. The client_assertion, import_do_not_edit values should not be edited, as they will be pre-populated once the previous values are defined. Follow the guidance provided at https://developer.va.gov/explore to obtain the necessary collection variable values for API calls. For other APIs, consult the Lighthouse CCG API page related to your desired interaction for accurate API-specific collection variable values. Some example values for the Collection variables following the Patient Health API will be shown below:
+
+a. aud (Collection variable)
+Example aud value:
+https://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token
+b. token_endpoint (Collection variable)
+Example API tokenUrl value:
+https://sandbox-api.va.gov/oauth2/health/system/v1/token
+c. clientId (Collection variable)- This value is obtained from signing up for sandbox access using the dev portal link
+Example clientId value:
+Your CCG clientId
+
```

Comment:
Remove extra new line here, so that it's similar to the other a, b, c . . .

Reply:
added consistent line breaks